README.md

```diff
@@ -12,19 +12,21 @@ See the support scope for [Red Hat Technology Preview](https://access.redhat.com
 ## Description
 
 The RHTPA service is the downstream redistribution of the [Trustification](https://github.com/trustification/trustification) project.
-The automation contained within this Git repository installs and configures the components of RHTPA to run on a single RHEL server, which uses a standalone containerized deployment.
-A Kubernetes-based manifest creates containers that uses [`podman kube play`](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
+The automation contained within this Git repository installs and configures the components of RHTPA to run on a single RHEL server, which uses a standalone containerized deployment. A Kubernetes-based manifest creates containers that uses [`podman kube play`](https://docs.podman.io/en/latest/markdown/podman-kube-play.1.html).
```

Comment:
I would suggest having one sentence per line, instead of having multiple sentences on one line, its better for readability and text searching.  For the first sentence, I suggest a slight rewrite at the end of the sentence:

`... single RHEL server, which uses a standalone containerized deployment.` --> `... single RHEL server by using a standalone containerized deployment.`

Reply:
applied